btrack/dataio.py

```diff
@@ -360,13 +364,13 @@ class HDF5FileHandler:
 
     def __init__(
         self,
-        filename: str,
+        filename: os.PathLike,
         read_write: str = "r",
         obj_type: str = "obj_type_1",
     ):
 
         self._f_expr = None  # DO NOT USE
-        self._object_type = None
+        self._object_type = ""
```

Comment:
I think you can just remove this line, as the next line will set `_object_type` (via. the setter)?

Reply:
Yep that looks correct